modules/tinymce/CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - The `mceInsertClipboardContent` command will now also accept a `files` property to insert images. #TINY-9776
 
 ### Fixed
+- Creating lists is possible when the selection is in an editable area inside a non-editable area. #TINY-10000
```

Comment:
Always past tense for 'fixed' items. And as past tense I think it reads better swapped around.

```suggestion
- When an editable area was nested inside a non-editable area, creating lists was not possible. #TINY-10000
```